scipy/stats/tests/test_stats.py

```diff
@@ -4452,14 +4452,26 @@ def test_normalitytests():
     # numbers verified with R: dagoTest in package fBasics
     st_normal, st_skew, st_kurt = (3.92371918, 1.98078826, -0.01403734)
     pv_normal, pv_skew, pv_kurt = (0.14059673, 0.04761502, 0.98880019)
+    st_skew_less, st_kurt_less = st_skew, st_kurt
```

Comment:
Not a big deal, but you could do away with these assignments. I don't recall running across any statistics that change based on the alternative being compared, so I don't think just comparing against `st_skew` would be too confusing.

Reply:
Done!